java/piranha/src/main/java/com/uber/piranha/config/Config.java

```diff
@@ -308,7 +347,8 @@ private static void requireType(String valK, Object v, Class<? extends Object> k
   private static void validateConfigOptsValue(String valK, Object v) {
     if (OPT_TESTS_CLEAN_BY_SETTERS_ENABLED.equals(valK)
         || OPT_TESTS_CLEAN_BY_SETTERS_IGNORE_OTHERS.equals(valK)
-        || ALLOW_METHOD_CHAIN.equals(valK)) {
+        || ALLOW_METHOD_CHAIN.equals(valK)
```

Comment:
Is there a possibility where for certain methods, method chains are allowed and for the remaining invocation as argument is allowed?

Reply:
Yes it is possible. Actually that scenario is covered in [properties_method_chain.json](https://github.com/uber/piranha/pull/165/files/15d1b654e160281d3ee89193164d4b18050682ac#diff-09ae089d001f6a6b6cdd5941367e2cd2927f005c5cbba8d04c4396f8ae35c25e)[](https://sourcegraph.com/github.com/uber/piranha/-/compare/6ff186008d50640ebb39689d77207c7a6cd2cfc2...15d1b654e160281d3ee89193164d4b18050682ac?utm_source=chrome-extension#diff-b25b21f387bdd6995e5b37769002a94a)